ext/telemetry/errors.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+module.exports = function(tracker) {
+  return {
+    trackUncaught: () => {
+      window.onerror = (msg, url, line, col, err) => {
```

Comment:
I think we should rather use [`process.on('uncaughtException', err => ...)`](https://nodejs.org/api/process.html#process_event_uncaughtexception)as I think it's what atom is using in its exception reporter.
But if we still want to use `window.onerror` then we probably should make sure that we store the previously defined value and restore it at the end.